src/pages/AdvertiseData/components/TradeForm.tsx

```diff
@@ -239,7 +239,7 @@ export const TradeForm: React.FC<TradeFormProps> = (props) => {
   };
 
   function validateBaseInput() {
-    if (!dataNFTStreamUrl.includes("https://") || !dataNFTPreviewUrl.includes("https://") || !dataNFTMarshalService.includes("https://")) {
+    if (!dataNFTPreviewUrl.includes("https://") || !dataNFTMarshalService.includes("https://")) {
```

Comment:
This should be changed to !(dataNFTPreviewUrl.includes("https://") || dataNFTPreviewUrl.includes("ipns://"))

Reply:
A better check would be
```suggestion
     dataNFTStreamUrl.startsWith("https://") || dataNFTStreamUrl.startsWith("ipns://")
```